docs/3.0.0-beta.x/guides/registering-a-field-in-admin.md

```diff
@@ -0,0 +1,214 @@
+# Creating a new Field in the administration panel
```

Comment:
Are you creating a new filed or you are replacing the current WYSIWYG?
Because at the end your are using the existing `RichText` filed type, so you didn't create a fresh new field?

Reply:
Replacing one by creating a new